include/class.orm.php

```diff
@@ -2567,12 +2567,35 @@ function compileQ(Q $Q, $model, $parens=true) {
             }
             // Handle simple field = <value> constraints
             else {
+                if (is_string($field) && strpos($field, 'count') !== false) {
```

Comment:
This is a no-no! Think of ORM as framework code, it shouldn't have application logic at all.